### PR TITLE
Updating external-dns image version from 0.8.0 to 0.10.2

### PR DESCRIPTION
**Pull request by Connect2naga**

### CODE REVIEW

**Comment by lmzuccarelli** [comment]
@Connect2naga - can you investigate why the tests are failing

File: config/manager/kustomization.yaml [hunk 1]
```diff
@@ -8,5 +8,5 @@ generatorOptions:
   disableNameSuffixHash: true
 images:
 - name: controller
-  newName: controller
+  newName: quay.io/snagarju/external-dns-operator
```
[the remark this answers is not in the document]
Comment (Connect2naga) [reply config/manager/kustomization.yaml]: thanks , removed :)

**Comment by lmzuccarelli** [comment]
@Connect2naga  - good to see all tests passing

File: config/manager/kustomization.yaml (hunk 1 above)
[the remark this answers is not in the document]
Comment (lmzuccarelli) [reply config/manager/kustomization.yaml]: @Connect2naga - can I ask why you have a reference to your personal repo ?

[the remark this answers is not in the document]
Comment (Connect2naga) [reply config/manager/kustomization.yaml]: @lmzuccarelli :  while testing in local, the `make deploy` will do this change.
so, missed this clean up and latest push it removed..

**Pull request merged**